tests/test_global.py

```diff
@@ -32,11 +32,13 @@ def check_invalid_recipes(recipes, invalids, loaded):
     for recipe in recipes:
         assert recipe
         assert re.search("(yml|yaml)$", recipe)
```

Comment:
Comment should be removed.

Reply:
done https://github.com/buildtesters/schemas/pull/34/commits/d26f0a00793c8999123f3c49c7b42e3bf5fc34d0